ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/teleoperation/RDXTeleoperationManager.java

```diff
@@ -584,12 +657,48 @@ private void process3DViewInput(ImGui3DViewInput input)
    public void renderImGuiWidgets()
    {
       hardwareControlStateManager.renderImGuiWidgets();
+      ImGui.sameLine();
+      if (ImGui.button(labels.get("N-Pose")))
+         goNPose();
       trajectoryTimeSlider.renderImGuiWidget();
       renderWholeBodyWidgets();
       locomotionManager.renderImGuiWidgets();
       armManager.renderImGuiWidgets();
    }
 
+   private void goNPose()
+   {
+      double trajectoryTime = 3.0;
+
+      for (RobotSide side : RobotSide.values)
+         armManager.executeArmAngles(side, PresetArmConfiguration.HOME, trajectoryTime);
+
+      if (robotModel.getRobotVersion().hasHead())
+      {
+         NeckJointName[] neckJointNamesArray = syncedRobot.getRobotModel().getJointMap().getNeckJointNames();
+         double[] desiredNeckJointValues = new double[neckJointNamesArray.length];
+         for (int i = 0; i < neckJointNamesArray.length; i++)
+         {
+            desiredNeckJointValues[i] = 0.0;
```

Comment:
I think this is an Alex specific parameter. Would be good to at least add a comment stating that, or a TODO to extract a parameter